supporting_scripts/code-coverage/per_module_cov_report/parse_module_coverage.py

```
@@ -0,0 +1,75 @@
+import os
+import xml.etree.ElementTree as ET
+import argparse
+
+def get_report_by_module(input_directory):
+    results = []
+    for module_folder in os.listdir(input_directory):
+        module_path = os.path.join(input_directory, module_folder)
+
+        if os.path.isdir(module_path):
+            report_file = os.path.join(module_path, f"jacocoTestReport.xml")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)